source/brailleDisplayDrivers/alva.py

```diff
@@ -304,14 +333,17 @@ def _handleTime(self, timeStr):
 		else:
 			log.debug("Time not synchronized. Display time %s"%displayDateTime.isoformat())
 
-	def _syncTime(self, dt):
+	def _syncTime(self, dt: datetime.datetime):
 		log.debug("Synchronizing braille display date and time...")
 		timeList = [

```

Comment:
Now, this really looks like a case where we might want to use a bytearray instead.